MapboxNavigation/UserCourseView.swift

```diff
@@ -36,6 +36,19 @@ public extension CourseUpdatable {
  A view representing the user’s location on screen.
  */
 public class UserPuckCourseView: UIView, CourseUpdatable {
+    private var lastLocationUpdate: Date?
+    private var staleTimer: Timer!
+    
+    /// Time interval tick at which Puck view is transitioning into 'stale' state
+    public var staleRefreshInterval: TimeInterval = 1 {
+        didSet {
+            staleTimer.invalidate()
+            initTimer()
+        }
+    }
+    /// Time interval, after which Puck is considered 100% 'stale'
+    public var staleInterval: TimeInterval = 60
```

Comment:
I thought that it could be handy to make these public. Maybe we should also allow explicit way to "turn off" puck staling?